compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropView.skiko.kt

```diff
@@ -16,6 +16,51 @@
 
 package androidx.compose.ui.viewinterop
 
-// Interop Views aren't supported in Skiko. Use Any instead of Nothing because we can't have a type
-// aliased to Nothing appear as a function return type or property type.
-actual typealias InteropView = Any
\ No newline at end of file
+import androidx.compose.ui.InternalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.PointerEvent
+import androidx.compose.ui.input.pointer.PointerEventPass
+import androidx.compose.ui.input.pointer.PointerInputFilter
+import androidx.compose.ui.node.ModifierNodeElement
+import androidx.compose.ui.node.PointerInputModifierNode
+import androidx.compose.ui.unit.IntSize
+
+/**
+ * Modifier to associate an [InteropView] with a [PointerInputFilter] to allow hit testing it and
```

Comment:
A doc should be updated. No more `PointerInputFilter` + it's node, not just "Modifier"

Reply:
done